### ISSUE TITLE
sequence_output

### ISSUE THREAD

**Issue by qiunlp** [HKUST-KnowComp/C2#1]
joint_model/Model.py  line 230 
sequence_output, _ = self.encoder(input_ids=batch_of_segments, attention_mask=token_masks) 

When I debug, I find that:
sequence_output = "last_hidden_state"

So, I can't get "start_representation", and **loss** always is 0

Thank you!
